cmd/skywire-cli/commands/node/gen-config.go

```
@@ -186,3 +189,19 @@ func defaultSkyproxyClientConfig() visor.AppConfig {
 		Port:      routing.Port(skyenv.SkyproxyClientPort),
 	}
 }
+
+func getLocalIPAddress() string {
+	addrs, err := net.InterfaceAddrs()
+	if err != nil {
+		log.WithError(err)
+	}
+
+	for _, a := range addrs {
+		if ipnet, ok := a.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
+			if ipnet.IP.To4() != nil {
+				return ipnet.IP.String() + ":7777"
+			}
+		}
+	}
+	return ""
```

I would return an error in case an address cannot be found.